<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamDialog</class>
 <widget class="QDialog" name="ExamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>41</y>
     <width>671</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="questionLabel">
        <property name="text">
         <string> 题目内容：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="answerWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="prevButton">
          <property name="text">
           <string>上一题</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="text">
           <string>提交试卷</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>下一题</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
